build/index.js

```diff
@@ -0,0 +1,117 @@
+"use strict";
```

Comment:
Hm I don't think built files should be versioned. My understanding with ts libraries is that they publish the built files to npm but only version ts files

Reply:
correct done